### ISSUE TITLE
aXe Coconut issue for rule: Images must have alternate text

### ISSUE THREAD

**Issue by marinho** [dequelabs/axe-core#762]
Even though my element **does have** `alt` with a value, it still says it doesn't:

```
<svg focusable="false" role="img" alt="error" style="user-select: text;">
  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/assets/css/images/icons.svg#icon-error" style="user-select: text;"></use>
</svg>
```

Remarks:

- There's no Shadow DOM involved.
- The `user-select: text` is present because StencilJS does so (as a web component)
- The parent component is a `role="alert"` and such image is the icon representing its message type (i.e. warning, confirmation, error, success, etc.).

**Comment by WilcoFiers** [comment]
Thanks for reporting. `alt` is not a valid attribute for SVG, which is why aXe-core is flagging this. You should use `aria-label` instead. The problem here is that aXe-core is misreporting what you can do to resolve it - adding an alt attribute doesn't fix this. This is a known bug, which got fixed recently in #683. We'll release an update to Coconut soon which has the change in it.
